optimize/upgrade/src/main/java/io/camunda/optimize/upgrade/plan/factories/Upgrade313To86PlanFactory.java

```diff
@@ -65,6 +68,11 @@ private static List<DeleteIndexIfExistsStep> deleteProcessInstanceArchiveIndexIf
         .toList();
   }
 
+  private static UpdateIndexStep removeCombinedReportFieldFromProcessReportIndex() {
+    return new UpdateIndexStep(
+        new SingleProcessReportIndexES(), "ctx._source.remove(\"combined\");");
+  }
```

Comment:
You mean the decision report index? Did not bother too much with decision report adjustments because those will be removed too. I had started working on that removal but then paused it to wait for the OS changes to be merged because I noticed that there are a bunch of structural changes that could be causing conflicts.

Reply:
I was anticipating for the Opensearch changes to be merged soon, but if it will be delayed longer I might open a PR with just the migration bits for the index changes.

Reply:
I did mean decision, yeah - sorry